app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/SelectSongScreen.kt

```diff
@@ -81,3 +89,24 @@ fun SelectSongScreen(navActions: NavigationActions, viewModel: SpotifySongsActio
             }
       }
 }
+
+// TODO :Merge with @Joriba stuff if possible
```

Comment:
For this, idk if this is possible to maybe merge with your previous work? (if not it's fine, just remove the TODO)